lib/beebee/storage/redis.ex

```diff
@@ -49,6 +49,22 @@ defmodule BeeBee.Storage.Redis do
     end
   end
 
+  @impl true
+  def delete(short_tag) do
+    case delete_result(short_tag) do
+      {:ok, [1, 1]} ->
+        {:ok}
```

Comment:
```suggestion
        :ok
```

Reply:
@codeadict thank you for the feedback! This PR is still a work in progress, please review when it's ready.